### PR TITLE
SRE-6066

### PR DESCRIPTION
**Pull request by pedropaccola**

### CODE REVIEW

File: spinner/build.sh [hunk 1]
```diff
@@ -48,15 +48,29 @@ function build_service_liferay {
 
 	cp ../../orca/templates/liferay/resources/usr/local/liferay/scripts/pre-startup/10_wait_for_dependencies.sh build/liferay/resources/usr/local/liferay/scripts/pre-startup
 
-	(
-		echo "FROM $(grep -e '^liferay.workspace.docker.image.liferay=' "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/liferay/gradle.properties" | cut -d'=' -f2)"
+	if [ "$(git --git-dir "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/.git" rev-parse --abbrev-ref HEAD)" == "master" ]
```
Comment (pedropaccola) [inline spinner/build.sh:51]: Using --git-dir directive, unfortunately the flags can't be reordered.

Comment (natocesarrego) [reply spinner/build.sh:51]: @pedropaccola, did you mean that we can't do the following, right?
```
git --abbrev-ref HEAD --git-dir "${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/.git" rev-parse)
```

File: spinner/build.sh [hunk 2]
```diff
@@ -428,16 +424,16 @@ function check_usage {
 }
 
 function get_environment_descriptor {
-	local file="${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/automation/environment-descriptors/${LXC_ENVIRONMENT}.json"
+	local descriptor="${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/automation/environment-descriptors/${LXC_ENVIRONMENT}.json"
```
Comment (natocesarrego) [inline spinner/build.sh:427]: @pedropaccola, can you please rename to `environment_descriptor` to keep the consistency with the method's name?

Thanks

Comment (pedropaccola) [reply spinner/build.sh:427]: Will do! Thanks @natocesarrego

File: spinner/build.sh (hunk 1 above)
Comment (pedropaccola) [reply spinner/build.sh:51]: Right! Both `-C` and `--git-dir` are flags for the `git` command, while `--abbrev-ref` is a flag for `rev-parse` command.

**Pull request closed without merging**